chart/values.yaml

```diff
@@ -20,6 +20,10 @@ crawler_logging_opts: "stats,behaviors,debug"
 # to enable, set to a value other than 'false'
 crawler_extract_full_text: false
 
+# use sitemap
+# to enable, set to value true
+crawler_use_sitemap: false
```

Comment:
Hm, do we want to add this here as well? I think the CRAWL_ARGS will override the config.yaml settings, which is where the user-configured options will go. I think we probably shouldn't also add it here and in configmap, unless we want to override the user setting..

Reply:
I've added a comment explaining what setting this to `true` will mean (it will apply the sitemap option for all seeded crawls, though it doesn't have an effect on URL lists due to the limit in place).

Unless I'm missing something, with the way the conditional works this value has to be in `chart/values.yaml` for the option to work at all.

Reply:
Removed! I had overcomplicated it :)